src/love.cpp

```diff
@@ -141,6 +141,17 @@ enum DoneAction
 	DONE_RESTART,
 };
 
+static void print_usage()
+{
+	printf("usage:\n"
```

Comment:
I wonder if it makes sense to link to the homepage or wiki, after the usage examples?

Reply:
Added one sentence from the homepage and a link